.github/workflows/app.yml

```diff
@@ -0,0 +1,45 @@
+name: Test and build Jellyfin Android
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Gradle wrapper and dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-java-${{ matrix.java }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
```

Comment:
is `${{ matrix.java }}` still set? It looks to me as if u don't have a matrix obj in this workflow and only build with JDK 11 anyway?

Reply:
Lazy copy-pasting on my end, and you expose me just like that. :eyes:
Thanks for the heads up, gonna fix!